src/main/kotlin/command/Timeout.kt

```
@@ -0,0 +1,99 @@
+        if (member.isOwner) {
+            throw CommandExitException("You cannot timeout the owner")
+        }
+        try {
+            when (unit) {
+                "seconds" -> member.timeoutFor(time.toLong(), TimeUnit.SECONDS).queue()
+                "minutes" -> member.timeoutFor(time.toLong(), TimeUnit.MINUTES).queue()
+                "hours" -> member.timeoutFor(time.toLong(), TimeUnit.HOURS).queue()
+                "days" -> member.timeoutFor(time.toLong(), TimeUnit.DAYS).queue()
+                else -> throw CommandExitException("Invalid unit")
+            }
+        } catch (e: IllegalArgumentException) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)